src/pages/reference/python/v0/websocket/websocket-on.mdx

```diff
@@ -0,0 +1,61 @@
+export const description =
+  "Reference for Nitric's Python library - Register a handler for Websocket events."
+
+# Python - websocket.on()
+
+Register a handler for a specific websocket event.
+
+```python
+from nitric.resources import websocket
+from nitric.application import Nitric
+
+chat_websocket = websocket("chat")
+
+@chat_websocket.on("connect")
+async def on_connect(ctx):
+    print("connecting ", ctx.req.connection_id)
+
+    ctx.res.success = true
+
+Nitric.run()
+```
+
+## Parameters
+
+<Properties>
+  <Property name="event_type" required type="string">
+    The event to respond to on the websocket, valid values are 'connect',
+    'disconnect' and 'message'.
```

Comment:
```suggestion
    The event to respond to on the websocket, valid values are 'connect', 'disconnect' and 'message'.
```

The docs appear to be respecting this line break, let's remove it since it causes a break in a strange location on the docs site.

<img width="877" alt="Python_-_websocket_on___-_Nitric_Documentation" src="https://github.com/nitrictech/docs/assets/4121391/f7bdf3ba-9f53-4b97-b710-34bb71be0967">